recipes/BEHST/meta.yaml

```diff
@@ -0,0 +1,59 @@
+package:
+  name: behst
+  version: 29efb4c3a8e333a2824430ef32d8bf01
```

Comment:
The goal is to have users be able to get the most up-to-date version of the package, which in turn means conda needs to be able to figure out what the latest version is. [PEP440](http://legacy.python.org/dev/peps/pep-0440/) is generally a good guide to follow if you haven't decided on versions yet; `0.1` would be a better version than `tag_for_conda2`.  Then you would modify the source URL to reflect the new tag and use that tag as your version.